imports/ui/statistics/PlayerModal.js

```diff
@@ -0,0 +1,28 @@
+import Players from '../../api/Players';
+import Constants from '../../api/Constants';
+
+import './PlayerModal.html';
+
+Template.PlayerModal.helpers({
+	getPlayerInfo() {
+		let player = Session.get("selectedStatistics");
+		if (player) {
+			let info = {};
+			info.name = player.leagueName;
+			info.elo = player.elo.toFixed(3);
+			info.winRate = (player.handsWin / player.handsTotal * 100).toFixed(1);
```

Comment:
Maybe we can keep it as is for now, and change it later if it makes a large enough difference. It's only a matter of changing 1 to 2, after all.